lxd/backup.go

```diff
@@ -399,3 +384,76 @@ func backupCreateTarball(s *state.State, path string, backup backup) error {
 
 	return nil
 }
+
+func pruneExpiredContainerBackupsTask(d *Daemon) (task.Func, task.Schedule) {
+	f := func(ctx context.Context) {
+		pruneExpiredContainerBackups(ctx, d)
+	}
+
+	pruneExpiredContainerBackups(context.Background(), d)
+
+	first := true
+	schedule := func() (time.Duration, error) {
+		interval := 24 * time.Hour
```

Comment:
Pretty sure we'll want to run this much more frequently, every hour would be a good start I'd think.